taichi/backends/vulkan/vulkan_api.h

```diff
@@ -41,6 +44,10 @@ struct VulkanQueueFamilyIndices {
   bool is_complete() const {
     return compute_family.has_value();
   }
+
+  bool is_complete_for_ui() {
```

Comment:
nit: Later we can change this to take in bitmask, e.g. `VK_QUEUE_GRAPHICS_BIT`, and check if `VulkanQueueFamilyIndices` satisfies that

Reply:
Later as in after we only set `VK_QUEUE_COMPUTE_BIT` for our compute queue?  IIUC, the reason why we currently set both `VK_QUEUE_COMPUTE_BIT` and `VK_QUEUE_GRAPHICS_BIT` is that we need them for debugging? @bobcao3 

If thats the case, after GGUI, I can create a separate PR that

1. removes the debug swapchain
2. only use `VK_QUEUE_COMPUTE_BIT` for the compute queue
3. fix this nits.

Reply:
Sounds good by me :-)